### PR TITLE
feat: add new minValue and maxValue to GroupSnapshotMetadata Properties

### PR DESCRIPTION
**Pull request by MartinGbz**

### CODE REVIEW

File: src/migration/02172023/migrate-groups-dynamodb-02172023.migration-test.ts [hunk 1]
```diff
@@ -225,16 +226,26 @@ describe("Test migration", () => {
 const computeProperties = (data: FetchedData): Properties => {
   const valueDistribution: { [tier: number]: number } = {};
   let accountsNumber = 0;
+  let minValue: BigNumberish = ethers.constants.MaxUint256;
```
Comment (leosayous21) [inline src/migration/02172023/migrate-groups-dynamodb-02172023.migration-test.ts:229]: let just put null by default

File: src/migration/12212022/migrate-groups-dynamodb-12212022.ts [hunk 2]
```diff
@@ -62,17 +63,27 @@ export const migrateGroupsProperties = async ({
 const computeProperties = (data: FetchedData): Properties => {
   const valueDistribution: { [tier: number]: number } = {};
   let accountsNumber = 0;
+  let minValue: BigNumberish = ethers.constants.MaxUint256;
```
Comment (leosayous21) [inline src/migration/12212022/migrate-groups-dynamodb-12212022.ts:66]: same initiate by default to null for min and max

File: src/topics/group-generator/group-generator.ts [hunk 3]
```diff
@@ -284,6 +299,20 @@ export class GroupGeneratorService {
             : ({
                 accountsNumber: Object.keys(group.data).length,
                 valueDistribution: { 1: Object.keys(group.data).length },
+                minValue: Object.values(group.data)
```
Comment (leosayous21) [inline src/topics/group-generator/group-generator.ts:302]: all the logic should be in "computeProperties". Filter "group.name === "sismo-contributors"" there

File: src/topics/group-generator/group-generator.ts [hunk 4]
```diff
@@ -257,6 +258,20 @@ export class GroupGeneratorService {
...
```
Comment (leosayous21) [inline src/topics/group-generator/group-generator.ts:261]: should be in compute properties (with accountsNumber and value distribution)

File: src/migration/02172023/migrate-groups-dynamodb-02172023.migration-test.ts (hunk 1 above)
Comment (MartinGbz) [reply src/migration/02172023/migrate-groups-dynamodb-02172023.migration-test.ts:229]: But a BigNumberish can't be null, so `minValue` and `maxValue` will be of type: `BigNumberish | null`
that's why I thought put 0 and MaxUint256 is better.

**Review by yum0e — APPROVED** [review]

**Pull request merged**